dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -166,3 +172,10 @@ def update(self, df_series):
         self._update_helper(df_series, profile)
 
         return self
+
+    def counts(self):
+        """
+        Returns the dict of categories with the number of occurrences
+        for each category.
+        """
+        return dict(self._categories.items())
```

Comment:
is this needed? if we wanted this, i would suggest using the code in the profile
```python
 dict(
    sorted(self._categories.items(), key=itemgetter(1),
                reverse=True)[:num_top_categories])
```
but have an input arg for the num_top categories and then the profile could call:
`self.category_counts(top_k_categories=5)`